### ISSUE TITLE
Realtime reporting functions incomplete

### ISSUE THREAD

**Issue by WillemPaling** [randyzwitch/RSiteCatalyst#185]
Just an FYI – I'm working on it with a colleague. I'll create a new branch and submit a PR.

**Comment by randyzwitch** [comment]
Great, looking forward to it. Is this the same as #76 ?

**Comment by WillemPaling** [comment]
Aah, I thought it totally wasn't working. But it's more of a doco issue. You can request any realtime report you like, but it'll only work if it fits within the defined reports.

I'll update doco, and have a go at #76.

**Comment by jdeboer** [comment]
Hey guys. Is there anything under development to extend the real-time reporting capabilities to more than one dimension? I'm hoping to use this for demoing real-time prediction.

**Comment by WillemPaling** [comment]
Hi Johan, have you tried? Should be too hard to parse the response. I'd say it'd be a few hours to make it work.

**Comment by jdeboer** [comment]
The RSiteCatalyst package currently only allows one dimension to be reported in realtime reports. Was hoping that there might be something being worked on already to address the limitation. Might have to leave this for another time as I'm in the final week of the project and have a few other things to sort out first. Thanks anyway.
